Dockerfile.mysql

```diff
@@ -0,0 +1,8 @@
+FROM mysql:8.0
+
+ENV MYSQL_ROOT_PASSWORD=rootpassword
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=паролем1234
```

Comment:
The `MYSQL_PASSWORD` contains non-ASCII characters. It's recommended to use ASCII characters for environment variables to avoid potential compatibility issues.